### PR TITLE
Make getKey and selection props mutually exclusive

### PR DESCRIPTION
**Pull request by okonech**
## Current Behavior

DetailsList allows passing the "getKey" prop along with a "selection" prop. This is confusing to users, since the "getKey" prop function will never be called. Instead, the Selection object passed in the "selection" prop either explicitly declares a getKey function, or uses the default getKey implementation (item) => item.key. 
The function passed in the "getKey" prop is only used when a Selection object is not passed in the "selection" prop. In this case, DetailsListBase creates a new internally passed Selection object which is passed the function from "getKey" prop.

## User Impact
Items may not receive proper selection when loading new data. Items may not retain checked status when the Detailslist is sorted

## Workaround
Do not pass the getKey function if the Selection object is passed. Users must instead define the getKey function on the Selection object.

## New Behavior

DetailsList will warn uses in non-production environments when "getKey" and "selection" props are passed simultaneously. Making "getKey" and "selection" mutually exclusive DetailsList properties prevents users from making this mistake.

## Related Issue(s)

(https://github.com/microsoft/fluentui/issues/23614)

Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3a4adc9678288e62d86487b84c6e3a4ce55cfae5'>3a4adc9678288e62d86487b84c6e3a4ce55cfae5</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/BaseButton-mount_base.html">781</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/BaseButton-mount.html">777</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Breadcrumb-mount_base.html">2372</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Breadcrumb-mount.html">2363</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Checkbox-mount_base.html">2226</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Checkbox-mount.html">2213</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/CheckboxBase-mount_base.html">1940</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/CheckboxBase-mount.html">1936</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ChoiceGroup-mount_base.html">4070</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ChoiceGroup-mount.html">4104</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ComboBox-mount_base.html">844</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ComboBox-mount.html">840</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/CommandBar-mount_base.html">9059</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/CommandBar-mount.html">9095</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ContextualMenu-mount_base.html">10230</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ContextualMenu-mount.html">10076</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DefaultButton-mount_base.html">981</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DefaultButton-mount.html">976</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DetailsRow-mount_base.html">3308</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DetailsRow-mount.html">3294</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DetailsRowFast-mount_base.html">3275</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DetailsRowFast-mount.html">3256</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DetailsRowNoStyles-mount_base.html">3142</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DetailsRowNoStyles-mount.html">3139</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Dialog-mount_base.html">2413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Dialog-mount.html">2411</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DocumentCardTitle-mount_base.html">159</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/DocumentCardTitle-mount.html">160</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Dropdown-mount_base.html">2860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Dropdown-mount.html">2863</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/FocusTrapZone-mount_base.html">1612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/FocusTrapZone-mount.html">1620</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/FocusZone-mount_base.html">1523</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/FocusZone-mount.html">1570</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/IconButton-mount_base.html">1498</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/IconButton-mount.html">1524</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Label-mount_base.html">291</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Label-mount.html">303</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Layer-mount_base.html">2529</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Layer-mount.html">2509</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Link-mount_base.html">417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Link-mount.html">400</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/MenuButton-mount_base.html">1253</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/MenuButton-mount.html">1281</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/MessageBar-mount_base.html">1799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/MessageBar-mount.html">1816</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Nav-mount_base.html">3076</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Nav-mount.html">2837</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/OverflowSet-mount_base.html">921</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/OverflowSet-mount.html">924</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Panel-mount_base.html">1833</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Panel-mount.html">1852</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Persona-mount_base.html">874</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Persona-mount.html">893</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Pivot-mount_base.html">1246</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Pivot-mount.html">1245</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/PrimaryButton-mount_base.html">1103</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/PrimaryButton-mount.html">1108</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Rating-mount_base.html">6625</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Rating-mount.html">6679</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/SearchBox-mount_base.html">1135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/SearchBox-mount.html">1118</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Shimmer-mount_base.html">2165</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Shimmer-mount.html">2120</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Slider-mount_base.html">1659</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Slider-mount.html">1673</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/SpinButton-mount_base.html">4320</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/SpinButton-mount.html">4321</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Spinner-mount_base.html">369</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Spinner-mount.html">383</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/SplitButton-mount_base.html">2716</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/SplitButton-mount.html">2747</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Stack-mount_base.html">479</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Stack-mount.html">467</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/StackWithIntrinsicChildren-mount_base.html">1991</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/StackWithIntrinsicChildren-mount.html">1995</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/StackWithTextChildren-mount_base.html">4556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/StackWithTextChildren-mount.html">4493</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/SwatchColorPicker-mount_base.html">9969</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/SwatchColorPicker-mount.html">10036</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/TagPicker-mount_base.html">2487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/TagPicker-mount.html">2342</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/TeachingBubble-mount_base.html">80167</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/TeachingBubble-mount.html">80402</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Text-mount_base.html">369</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Text-mount.html">363</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/TextField-mount_base.html">1211</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/TextField-mount.html">1186</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ThemeProvider-mount_base.html">1008</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ThemeProvider-mount.html">1015</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ThemeProvider-virtual-rerender_base.html">559</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ThemeProvider-virtual-rerender.html">575</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1630</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1627</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Toggle-mount_base.html">672</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/Toggle-mount.html">684</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/buttonNative-mount_base.html">110</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24048/perf-test/buttonNative-mount.html">115</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by okonech** [comment]
@spmonahan Could you please take a look at this?

**Comment by spmonahan** [comment]
@okonech Thanks for the PR and sorry it's taken me a bit to look at it.

I agree that the existing behavior is confusing but this PR is a breaking API change so we cannot accept it as-is.

In reviewing your PR I looked into `Selection`'s implementation came across [this comment in Selection.ts](https://github.com/microsoft/fluentui/blob/master/packages/utilities/src/selection/Selection.ts#L11-L12):

```ts
/** Custom logic to generate item keys. Required if `TItem` does not have a `key` property. */
  getKey?: (item: TItem, index?: number) => string | number;
```

Of course this is far from ideal because not only do you have to read the Fluent source code to be aware of it, but it's also counterintuitive that you'd need to provide `getKey()` in two places. At a minimum we can do better by demonstrating the correct usage for Selection and getKey in the docs.

Perhaps there is also an opportunity to change [the way selection is initialized in `DetailsList` instead of changing the API](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/DetailsList/DetailsList.base.tsx#L806-L812)?

What do you think?

cc @behowell

**Review by spmonahan — CHANGES_REQUESTED** [review]
This change is a breaking API change. Putting a block on merging it while other options are discussed.

**Comment by okonech** [comment]
This gets a little wonky, but the simplest non-breaking change would be a new method for the Selection object:  setGetKey(getKey: (item: TItem, index?: number) => string | number) : void. Then in the referenced code, we would update the passed selection object with the passed getKey implementation.

It would be cleaner to instead spread the props of the Selection object passed to DetailsList and create a new Selection object (plus the DetailsList's implementation of getKey if a getKey was not provided)

`this._selection =
      props.selection ||
      new Selection({...[spread props.selection values], getKey: props.getKey});`

 but that might break use cases where the users rely on their manually created Selection object's reference being passed in some way. What do you think?

**Comment by okonech** [comment]
@spmonahan do you have any thoughts?

**Comment by okonech** [comment]
@spmonahan I've made the change to revert the breaking types change. The PR is now limited to warning users when they use the selection and getKey properties as they are intended to be mutually exclusive. This also fixes the recommended storybook example to follow suit.

**Review by spmonahan — APPROVED** [review]

**Comment by okonech** [comment]
@ThomasMichon could you please take a look at this?

**Comment by okonech** [comment]
@ThomasMichon ping :)

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by okonech** [comment]
@ThomasMichon could you please review?

File: change/@fluentui-react-519e3c9c-d7fd-40f6-b9f7-6a2d2e1af28f.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-519e3c9c-d7fd-40f6-b9f7-6a2d2e1af28f.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=24048&commitId=f4433ae2dc55f2b8fb80420cf282c0bfaa3ee2e6&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Review by ThomasMichon — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Pull request merged**